src/main/resources/google_checks.xml

```diff
@@ -19,6 +19,13 @@
 <module name="Checker">
   <module name="SuppressWarningsFilter"/>
 
+  <module name="SuppressWithNearbyTextFilter">
+    <property name="nearbyTextPattern"
+              value="@cs-: (\w+) for ([+-]\d+) lines"/>
```

Comment:
put it underhttps://github.com/checkstyle/checkstyle/blob/8433fa7168ad161d03f08287df2449ad5c8f3345/src/main/resources/google_checks.xml#L40 and  move SuppressWarningsFilter also to keep suppressions together.

Reply:
`"CHECKSTYLE.SUPPRESS\: (\w+) for ([+-]\d+) lines"`
to match https://github.com/checkstyle/checkstyle/blob/8433fa7168ad161d03f08287df2449ad5c8f3345/src/main/resources/google_checks.xml#L385C44-L385C77

Reply:
> put it underhttps://github.com/checkstyle/checkstyle/blob/8433fa7168ad161d03f08287df2449ad5c8f3345/src/main/resources/google_checks.xml#L40 and move SuppressWarningsFilter also to keep suppressions together.

done

https://github.com/checkstyle/checkstyle/blob/0386838241ac8d778427e82fb9e215a991ad6c4e/src/main/resources/google_checks.xml#L32-L47

also added `suppresswithnearbytextfilter.html` link above the suppression

Reply:
> `"CHECKSTYLE.SUPPRESS\: (\w+) for ([+-]\d+) lines"` to match https://github.com/checkstyle/checkstyle/blob/8433fa7168ad161d03f08287df2449ad5c8f3345/src/main/resources/google_checks.xml#L385C44-L385C77

done 

https://github.com/checkstyle/checkstyle/blob/0386838241ac8d778427e82fb9e215a991ad6c4e/src/main/resources/google_checks.xml#L42-L44